### ISSUE TITLE
Dev UI: Got two emails for server creation

### ISSUE THREAD

**Issue by sitapriyamoorthi** [getwilds/proof#21]
The Dev UI was running slow, so I deleted my old cromwell server (old = made using the prod PROOF UI).
I then recreated it using the Dev UI.
I got two email 1 minute apart saying that my server is ready. 
E-mail-1:
![Image](https://github.com/user-attachments/assets/82462df2-2777-4794-9c9b-56040960186d)
E-mail-2:
![Image](https://github.com/user-attachments/assets/0ff05b04-8855-41d1-9bf7-2b910e380de6)

I have only one server running however it says it proof-pr (which I imagine is production?)
![Image](https://github.com/user-attachments/assets/d44a01d1-b6be-4574-899a-578486653b09)

**Comment by sitapriyamoorthi** [comment]
I just recreated the same issue. I double checked that both prod and dev had no servers created. then i created a new server only using the dev and i still get two email!!! and the proof server says its proof_pr.. is this expected? should we be seeing proof-dev?
![Image](https://github.com/user-attachments/assets/4e9e7624-1dad-4524-aeef-7a1a822eb459)

**Comment by sckott** [comment]
Thanks Sita!

> I have only one server running however it says it proof-pr (which I imagine is production?)

prod and dev only exist for the shiny app and the proof api. and as dan said in slack both versions of the app point to the prod API. AFAIK there's no prod and dev cromwell servers.

**Comment by sckott** [comment]
@dtenenba thoughts on this? I think I've seen this once as well.

**Comment by dtenenba** [comment]
I believe you were working with the dev api using proofr. Starting a server in that way will generate an email, and so will starting a server in prod (with the shiny app, either dev or prod version). 
The two emails you got were sent about half an hour apart. I don't think they are related.

**Comment by sitapriyamoorthi** [comment]
I agree it is weird that I got two emails. This did not happen this morning when I used prod to create the server. However it did happen when I used dev to create the server. 
I intend to test out the S3 issue using the proof instructions @dtenenba sent
FYI I totally get only prod version of the server is created irrespective of which App i use (prod or dev).
